### PR TITLE
Adding 32Bit support

### PR DESCRIPTION
**Pull request by Yonisha**

### CODE REVIEW

File: core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java [hunk 1]
```diff
@@ -157,11 +157,7 @@ private static boolean initNativeCode() {
      */
     private static boolean loadLibrary() throws IOException {
         String model = System.getProperty("sun.arch.data.model");
```
Comment (amnonshmsft) [inline core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java]: Regarding `String model = System.getProperty("sun.arch.data.model");` - would this be right property to check also for Java OpenJDK (Zulu) distribution?

File: core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java [hunk 2]
```diff
@@ -68,22 +67,23 @@
...
                 InternalLogger.INSTANCE.error("Jni connector is only used on Windows OS.");
                 return false;
             }
 
-            if (loadLibrary()) {
-                return initNativeCode();
-            } else {
-                InternalLogger.INSTANCE.error("Failed to load native dll, Windows performance counters will not be used.");
-            }
+            loadLibrary();
```
Comment (amnonshmsft) [inline core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java]: `loadLibrary` is a too generic name - please rename to `loadNativeLibrary`

File: core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java [hunk 3]
```diff
@@ -68,22 +67,23 @@
...
+            initializationResult = initNativeCode();
         } catch (Throwable e) {
-            InternalLogger.INSTANCE.error("Failed to load native dll, Windows performance counters will not be used: '%s'.", e.getMessage());
+            InternalLogger.INSTANCE.error(
+                    "Failed to load native dll, Windows performance counters will not be used. " +
```
Comment (amnonshmsft) [inline core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java]: Alignment - first line (`"Failed to load...') should be 4 chars after `InternalLogger...`.
Second line `"Please make sure...` should be aligned with the first line.

File: core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java [hunk 4]
```diff
@@ -68,22 +67,23 @@
...
             }
 
-            if (loadLibrary()) {
-                return initNativeCode();
-            } else {
-                InternalLogger.INSTANCE.error("Failed to load native dll, Windows performance counters will not be used.");
-            }
+            loadLibrary();
+            initializationResult = initNativeCode();
         } catch (Throwable e) {
```
Comment (amnonshmsft) [inline core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java]: Why not initializing the native code as part of `loadNativeLibrary`?

File: core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java (hunk 1 above)
Comment (Yonisha) [inline core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java]: Good point.
I've verified it on zulu 1.6 64 bit JVM and it works as expected

**Pull request merged**